### PR TITLE
Add com.github.parnold_x.nasc (nasc-flatpak)

### PR DESCRIPTION
**Pull request by scx**
**NaSC** is an app where you do maths like a normal person. It lets you type whatever you want and smartly figures out what is math and spits out an answer on the right pane. Then you can plug those answers in to future equations and if that answer changes, so does the equations it's used in.
https://github.com/scx/flathub/tree/com.github.parnold_x.nasc
https://github.com/scx/nasc-flatpak

See also:
http://parnold-x.github.io/nasc
https://github.com/parnold-x/nasc
https://github.com/scx/nasc/tree/flatpak

### CODE REVIEW

**Comment by scx** [comment]
Upstream participation request:
https://github.com/parnold-x/nasc/issues/105

**Comment by scx** [comment]
Previous application submission:
https://github.com/flathub/flathub/pull/128

Changes compared to the previous application submission:
 * Update NaSC to the latest version
 * Update GNOME runtime to 3.30
 * Use `io.elementary.BaseApp`
 * Use `get_user_data_dir()` instead of hardcoded path
 * Enhance desktop file
 * Enhance AppData file
 * Add various patches to the build scripts
 * Port to Meson
 * Change Application ID (another attempt)

**Comment by scx** [comment]
testbot, build com.github.parnold_x.nasc

**Comment by scx** [comment]
@phw @AsciiWolf 
Are you interested in co-maintaining this package?

**Comment by AsciiWolf** [comment]
@scx Thanks for the offer, but I am probably not interested, sorry. I already maintain many Flathub apps.

**Comment by scx** [comment]
testbot, build com.github.parnold_x.nasc

**Comment by nedrichards** [comment]
bot, build com.github.parnold_x.nasc
testbot, build com.github.parnold_x.nasc

**Comment by scx** [comment]
Fails on arm (`cln`):
https://flathub.org/builds/#/builders/2/builds/8228

**Comment by nedrichards** [comment]
bot, build com.github.parnold_x.nasc

**Comment by scx** [comment]
Builds clean: https://flathub.org/builds/#/builders/2/builds/8229

**Comment by scx** [comment]
bot, build com.github.parnold_x.nasc

File: com.github.parnold_x.nasc.yaml [hunk 1]
```diff
@@ -0,0 +1,43 @@
+app-id: com.github.parnold_x.nasc
+base: io.elementary.BaseApp
+base-version: juno
+# Use GNOME instead of Freedesktop because of GtkSourceView
+runtime: org.gnome.Platform
+runtime-version: '3.32'
```
Comment (LeandroStanger) [inline com.github.parnold_x.nasc.yaml]: Update runtime to 3.34

**Comment by scx** [comment]
I've provided a single-file bundle for x86_64.
https://github.com/scx/nasc-flatpak/releases
https://github.com/scx/nasc-flatpak/releases/tag/v0.5.4+20191028

**Comment by scx** [comment]
bot, build com.github.parnold_x.nasc

**Comment by scx** [comment]
bot, build com.github.parnold_x.nasc

**Comment by scx** [comment]
Single-file bundle:
https://github.com/scx/nasc-flatpak/releases
https://github.com/scx/nasc-flatpak/releases/tag/v0.5.4+20191122

**Comment by scx** [comment]
bot, build com.github.parnold_x.nasc

**Comment by scx** [comment]
Single-file bundle:
https://github.com/scx/nasc-flatpak/releases
https://github.com/scx/nasc-flatpak/releases/tag/v0.5.4+20191123

File: com.github.parnold_x.nasc.yaml (hunk 1 above)
Comment (scx) [reply com.github.parnold_x.nasc.yaml]: When this application was submitted, the GNOME runtime was up to date. I've updated it to `3.32`, then to `3.34`. I see no point for further updates if this won't be merged soon.

**Pull request closed without merging**